1-exercises/B-while-loop/exercise.js

```diff
@@ -5,8 +5,24 @@
     The list of numbers should start with 0. n is being passed in as a parameter.
 */
 
-function evenNumbers(n) {
-    // TODO
+function evenNumbers(n)
+{
+    let total = "0";
+    let calc = 0;
+    
+    if (n === 0)
+    {
+        return console.log("");
+    }
+
+    for(let i = 1; i < n; i++)
```

Comment:
Hi Kerim, I've looked through your code and it's great! One suggestion I have here however is using a while loop instead of a for loop, e.g. while (i <= n).
Great work!